controllers/istiorevision/suite_test.go

```diff
@@ -89,8 +88,6 @@ var _ = BeforeSuite(func() {
 
 	err = k8sClient.Create(context.TODO(), &corev1.Namespace{ObjectMeta: v1.ObjectMeta{Name: operatorNamespace}})
 	Expect(err).NotTo(HaveOccurred())
-	err = os.Setenv("POD_NAMESPACE", operatorNamespace)
```

Comment:
The varable is only used to run the controller manager. These tests run the controller directly instead of running the manager.

Actually, this line here is exactly why I removed the global variable. If a controller needs to know the namespace, it should be passed in via a constructor argument.